library/entropy_poll.c

```diff
@@ -86,10 +92,15 @@ int mbedtls_platform_entropy_poll( void *data, unsigned char *output, size_t len
 
 /*
  * Test for Linux getrandom() support.
- * Since there is no wrapper in the libc yet, use the generic syscall wrapper
+ * When glibc > 2.25 include sys/random.h for getrandom(),
```

Comment:
Rather "When the C library is GNU libc and its version is greater than 2.25, include sys/random.h to use getrandom(), otherwise use the generic ..." ?

Reply:
fixed

Reply:
I don't see this change.

Reply:
sorry, it's my miss.
fixed now